config/samples/component-integration-installed/operator_v1alpha1_moduletemplate_skr-module.yaml

```diff
@@ -1,75 +0,0 @@
-apiVersion: operator.kyma-project.io/v1alpha1
```

Comment:
I believe that this should still be used in dual cluster setups / tests.

Reply:
clarified, not needed